src/components/Underline/index.tsx

```diff
@@ -11,7 +11,12 @@ const UnderlineSpan = styled.span`
 
 const Underline = () => {
   return (
-    <UnderlineSpan data-testid={"underline"} aria-label={"space for answer"} />
+    <UnderlineSpan
+      data-testid={"underline"}
+      // This is an empty box with a line under it, to indicate an answer would go here.
+      role="presentation"
+      title="An empty space to write an answer in"
```

Comment:
I'm a bit confused about this component -- if it's just an underline component, why have title or aria label?